apps/console/webpack.config.ts

```diff
@@ -126,6 +126,15 @@ module.exports = (config: WebpackOptionsNormalized, context: NxWebpackContextInt
                 authorizationCode: "<%=request.getParameter(\"code\")%>",
                 contentType: "<%@ page language=\"java\" contentType=\"text/html; charset=UTF-8\" " +
                     "pageEncoding=\"UTF-8\" %>",
+                // eslint-disable-next-line max-len
+                cookieproDomainScriptId: "<% String cookiepro_domain_script_id = System.getenv(\"cookiepro_domain_script_id\"); %>",
```

Comment:
Does this affect when we deploy the app on an external tomcat server?

Reply:
https://github.com/wso2/identity-apps/blob/master/docs/DEPLOYMENT.md#deploying-the-apps-on-an-external-server

Reply:
All Java Syntax used in the webpack plugin options are in-built Java methods. So, identity apps won't be affected with this.